templates/participants/partials/participant_experiments.html

```diff
@@ -0,0 +1,66 @@
+{% load form_tags %}
+
+<table class="min-w-full divide-y divide-gray-300">
+  <thead>
+    <tr>
+      <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Experiment</th>
+      <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Joined on</th>
+      <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Last message received at</th>
+    </tr>
+  </thead>
+  <tbody class="divide-y divide-gray-200" x-data="{experimentsRow: null}">
+    {% for experiment, info in experiment_data.items %}
+      {% with sessions=info|dict_lookup:"sessions" participant_data=info|dict_lookup:"participant_data" %}
+        <tr
+          class="cursor-pointer"
+          x-bind:class="experimentsRow === {{ forloop.counter }} ? 'bg-gray-300' : ''"
```

Comment:
you can do a striped table by adding the `table-zebra` class to the `table` element (https://v3.daisyui.com/components/table/#zebra)